AutoCollection/ServerRequestParser.ts

```diff
@@ -110,6 +106,10 @@ class ServerRequestParser extends RequestParser {
         return this.requestId;
     }
 
+    public getCorrelationContext() {
+        return this.correlationContext;
```

Comment:
Agreed. I'm open to suggestions; perhaps `getCorrelationContextHeader`? It is "whatever value is specified in the `correlation-context` header, if any"